python/ray/serve/http_proxy.py

```diff
@@ -54,11 +56,30 @@ async def _send_request_to_handle(handle, scope, receive, send):
 
     retries = 0
     backoff_time_s = 0.05
+    loop = asyncio.get_event_loop()
+    # We have received all the http request conent. The next `receive`
+    # call might never arrive; if it does, it can only be `http.disconnect`.
```

Comment:
is it possible to add a check for the http disconnect case?

Reply:
also, are we sure this is guaranteed?

Reply:
Yup. But I can add a check.